conformance/system_macros/flatten.ion

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test Cases:
+// flatten can be invoked by name, address, qualified address, qualified name, binary address, and binary system e-expression op code.
+// the argument values may be zero or more non-null lists and s-expressions
+// flatten expands to a stream whose elements are the concatenated elements of all its arguments
+// annotations on the argument (list and sexp) values are silently dropped
```

Comment:
This is more of a spec question, but might I wonder if there's a use case for allowing flatten to accept struct arguments and discard field names.

Reply:
@zslayton, thoughts? Or I could just open an issue for this.

Reply:
I think we need a use case to justify adding this and inform to the best behavior. Without one I'm inclined to just wait for an Ion 1.2.